README.md

```diff
@@ -9,19 +9,26 @@ Mount [Microsoft OneDrive][onedrive] storage as [FUSE] filesystem.
 
 ## Installation
 
-*Note: For Nix users, the program is already packaged via Nix Flake in `flake.nix`.*
+_Note: For Nix users, the program is already packaged via Nix Flake in `flake.nix`._
 
 1.  Use your package manager to install these dependencies:
     - pkg-config
     - openssl
     - fuse (libfuse)
+2.  On Debian 12 you may need to also install the following to resolve missing files:
 
-1.  Compile and install the program from crates.io:
+    - libssl-dev
+    - libfuse3-dev
+
+3.  Compile and install the program from crates.io:
 
     ```
     $ cargo install onedrive-fuse
     ```
 
+4.  If your rustc version isn't 1.65, and you don't mind not using the official repos, you can install it from here:
+    $ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
- Please refer to https://rustup.rs/ instead of copy-paste their command here. I think `curl | sh` without verification is a very bad practice. I refuse to endorse it.

- Also this step should go above previous `cargo` invocation.

Reply:
I have made all of the changes requested.